ui/src/pages/version/components/modal/EndpointsTableModelVersion.js

```diff
@@ -0,0 +1,132 @@
+/**
```

Comment:
Nit on the naming of these 2 files:
* EndpointsTableModelVersion.js -> ModelVersionEndpointsTable
* JobsTableModelVersion.js -> ModelVersionJobsTable

I feel that starting the file name with `ModelVersion...` is more descriptive of the thing that comes after (`JobsTable` / `EndpointsTable`).